controllers/replicationdestination.go

```diff
@@ -0,0 +1,92 @@
+package controllers
+
+import (
+	"fmt"
+
+	volsyncv1alpha1 "github.com/backube/volsync/api/v1alpha1"
+	"github.com/go-logr/logr"
+	pvcv1alpha1 "github.com/konveyor/volume-snapshot-mover/api/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+func (r *DataMoverRestoreReconciler) CreateReplicationDestination(log logr.Logger) (bool, error) {
+
+	// get datamoverrestore from cluster
+	dmr := pvcv1alpha1.DataMoverRestore{}
+	if err := r.Get(r.Context, r.NamespacedName, &dmr); err != nil {
```

Comment:
DMR CR will be created in the app ns, right? in that case should we change `r.NamespacedName` -> `r.req.NamespacedName` ?
cc: @shubham-pampattiwar